packages/palette/src/elements/Menu/Menu.tsx

```diff
@@ -80,7 +80,7 @@ export const MenuItem: React.FC<MenuItemProps> = ({
   ...props
 }) => {
   return (
-    <MenuLink href={href} {...props}>
+    <MenuLink href={href} {...props} hasLighterColor={Boolean(textColor)}>
```

Comment:
This makes sense!

Two questions:
- Is there a way to derive this from the `textColor` prop? (Or another prop?) Like-- if the text color is lighter than X, make the background hover color Y.
- I'm not super familiar with everywhere this component is used, but is it possible we want this change to be across-the-board and can avoid the conditional?

Reply:
@sweir27 I was trying not to introduce a new prop/litter this component with props for these one off things. 

I like the idea of inferring the background based off the `textColor`. I think we'd need to collaborate with the design team to get some specs for that. Could we create a ticket for this and I commit to revisiting during upkeep time?

Reply:
👍 Right now it looks like this prop will be `true` if _any_ `textColor` is provided to the component. That seems potentially confusing (especially if someone passes in `black100` for that value).

It might be worth following up quickly with the design team to see if they can provide some direction, or I'd rename this to `hasLighterTextColor` and accept the prop from the outermost component so it can be precisely controlled!

Reply:
I updated this PR to pass in `hasLighterTextColor` which is a boolean and is used to determine the background-color.